doc/examples/mr-example/pom.xml

```diff
@@ -61,17 +61,17 @@
     <dependency>
       <groupId>org.apache.avro</groupId>
       <artifactId>avro</artifactId>
-      <version>1.7.5</version>
+      <version>1.9.0</version>
```

Comment:
the idea here is that the docs example will work after the release of 1.9.0 right?

should we start updating the version here to match the current repo branch it's in?